### PR TITLE
add breadcrumbs prop to screen

### PR DESCRIPTION
**Pull request by acastro**

### CODE REVIEW

**Comment by mgollnick** [comment]
Looks good. Follows how tabs work.

File: src/components/screen/index.jsx [hunk 1]
```diff
@@ -62,6 +63,17 @@ const Screen = (props) => {
         {(header || primaryActions) &&
         <div className="screen-header-container">
           {header && <h1 className="screen-header-title">{header}</h1>}
+          {breadcrumbs.length > 0 &&
+          <Breadcrumb>
+            {breadcrumbs.map(tab => (
```
Comment (gdbrown) [inline src/components/screen/index.jsx]: @acastro call the var `breadcrumb` instead of tab

File: src/components/screen/index.jsx [hunk 2]
```diff
@@ -62,6 +63,17 @@ const Screen = (props) => {
         {(header || primaryActions) &&
```
Comment (gdbrown) [inline src/components/screen/index.jsx]: @acastro need to account for breadcrumbs being present.

File: src/components/screen/index.jsx [hunk 3]
```diff
@@ -109,6 +121,7 @@ Screen.propTypes = {
   footer: PropTypes.node,
   primaryActions: PropTypes.node,
   secondaryActions: PropTypes.node,
+  breadcrumbs: PropTypes.array,
```
Comment (gdbrown) [inline src/components/screen/index.jsx]: @acastro put breadcrumbs after `header` prop

**Pull request merged**